src/reply.rs

```diff
@@ -853,12 +858,13 @@ mod test {
     }
 
     impl super::ReplySender for AssertSender {
-        fn send(&self, data: &[&[u8]]) {
+        fn send(&self, data: &[IoSlice<'_>]) -> std::io::Result<()> {
             let mut v = vec![];
-            for x in bufs {
+            for x in data {
```

Comment:
Looks like this fixes a typo in the previous commit. Can you fix it there instead?